brooklinevoiceapp/mycity/test/integration_tests/test_nearest_police_station_intent.py

```diff
@@ -11,6 +11,8 @@
 
 MOCK_RESPONSE = test_constants.GET_POLICE_STATION_API_MOCK
 
+EXPECTED_RESPONSE = ps_intent.NO_RESULTS_RESPONSE
```

Comment:
I would rename this variable to be clearer since its at the file scope. This is the expected response when there is no response.

Reply:
Good point, done!